src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -579,6 +580,19 @@ public unsafe AssemblyInfo(MonoProxy monoProxy, SessionId sessionId, string url,
             Populate();
         }
 
+        public async Task<int> DebugId(MonoSDBHelper sdbAgent, CancellationToken token)
+        {
+            if (debugId != -1)
+                return debugId;
+            debugId = await sdbAgent.GetAssemblyId(Name, token);
```

Comment:
would it make sense to throw here if `debugId <= 0`?

Reply:
Do you mean after `GetAssemblyId`?

Reply:
yeah

Reply:
I will do it, but in my backport version of this PR I will not add the throw to avoid any side effect on net6, does it make sense?

Reply:
Okay, I changed my mind, I don't think we will get any side effect on net 6, because this is called from FindStaticTypeId and this has a try catch in the caller.